Kill weapons on drop, can no longer hide viewmodel

HymnsForDisco opened the pull request:
Will kill weapons immediately on drop, and the hide viewmodel feature has been disabled since applying effects to viewmodels no longer works in CS:GO

HymnsForDisco committed:
Kill weapons on drop, can no longer hide viewmodel

Will kill weapons immediately on drop, and the hide viewmodel feature has been disabled since applying effects to viewmodels no longer works in CS:GO

hlstriker commented:
Instead of removing all the code to allow 1 dropped weapon you should make it a cvar.

HymnsForDisco committed:
Allow dropped weapons as cvar

"weapons_allow_dropped_weapon" "1", "Whether to allow clients to drop weapons (max 1 per player)"

HymnsForDisco commented:
Added commit with cvar weapons_allow_dropped_weapon
